File: services/api/src/main/java/net/blueshell/api/service/EventPictureService.java
package net.blueshell.api.service;

import net.blueshell.api.base.BaseModelService;
import net.blueshell.api.exception.ResourceNotFoundException;
import net.blueshell.api.model.Event;
import net.blueshell.api.model.EventPicture;
import net.blueshell.api.model.EventSignUp;
import net.blueshell.api.model.User;
import net.blueshell.api.repository.EventPictureRepository;
import net.blueshell.api.repository.EventSignUpRepository;
import net.blueshell.api.service.brevo.EmailService;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Service;
import org.springframework.transaction.annotation.Transactional;
import sendinblue.ApiException;

import java.util.List;

@Service
public class EventPictureService extends BaseModelService<EventPicture, Long, EventPictureRepository> {

    @Autowired
    public EventPictureService(EventPictureRepository repository) {
        super(repository);
    }
}

File: api/src/main/kotlin/net/blueshell/api/feature/event/service/EventPictureService.kt
package net.blueshell.api.feature.event.service

import net.blueshell.api.feature.event.model.EventPicture
import net.blueshell.api.feature.event.repository.EventPictureRepository
import net.blueshell.api.shared.service.BaseModelService
import org.springframework.beans.factory.annotation.Autowired
import org.springframework.context.ApplicationEventPublisher
import org.springframework.stereotype.Service

@Service
class EventPictureService @Autowired constructor(
    repository: EventPictureRepository,
    events: ApplicationEventPublisher
) : BaseModelService<EventPicture, EventPicture.Id, EventPictureRepository>(repository)
